documentation/modules/exploit/multi/http/rails_web_console_v2_code_exec.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+ This module exploits an IP whitelist bypass vulnerability in the developer web console included with Ruby on Rails 4.0.x and 4.1.x. This module will also achieve code execution on Rails 4.2.x if the attack is launched from a whitelisted IP range.
+
+## Verification Steps
+
+**Prerequisites:**
+
+```
+$ gem install rails -v 4.2.6
```

Comment:
If youre going to give it in a code block, id remove the ```$``` so it can be copy pasted into a console